### PR TITLE
Improve rendering for tf.distribute.cluster_resolver.TPUClusterResolver in tpu_strategy API docs

### PR DESCRIPTION
**Pull request by 8bitmp3**
This should fix the auto-linking and rendering

image

### CODE REVIEW

File: tensorflow/python/distribute/tpu_strategy.py [hunk 1]
```diff
@@ -320,7 +320,7 @@ def __init__(self,
     """Synchronous training in TPU donuts or Pods.
 
     Args:
-      tpu_cluster_resolver: A tf.distribute.cluster_resolver.TPUClusterResolver,
+      tpu_cluster_resolver: A `tf.distribute.cluster_resolver.TPUClusterResolver`,
```
Comment (rxsang) [inline tensorflow/python/distribute/tpu_strategy.py:323]: A `tf.distribute.cluster_resolver.TPUClusterResolver` instance,

**Review by rxsang — APPROVED** [review]

**Review by rxsang — APPROVED** [review]

**Comment by gbaned** [comment]
@8bitmp3  Can you please address Ubuntu Sanity errors? Thanks!

**Comment by 8bitmp3** [comment]
> @8bitmp3 Can you please address Ubuntu Sanity errors? Thanks!

thanks @gbaned `exited with error code 1 ` - do you have an idea what it means? @rxsang

**Comment by rxsang** [comment]
> > Ubuntu Sanity errors? Thanks!
> 
> thanks

I don't think that's relevant, maybe a transient issue?

**Comment by 8bitmp3** [comment]
> > > Ubuntu Sanity errors? Thanks!
> > 
> > 
> > thanks
> 
> I don't think that's relevant, maybe a transient issue?

@gbaned @rxsang I think I found it in the logs: `2. do_pylint: Python 3 pylint  FAIL`

**Comment by 8bitmp3** [comment]
@gbaned @rxsang OK, PTAL, thank you

**Review by rxsang — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Pull request merged**